apps/academy-wep/components/learn/learning-module-card.tsx

```
@@ -0,0 +1,177 @@
+          </h3>
+          <p className="text-gray-600 mb-4 line-clamp-2 group-hover:text-gray-800 transition-colors duration-300 flex-grow">
+            {module.description}
+          </p>
+
+          {module.unlocked && (
+            <div className="flex items-center gap-4 mb-3">
+              <div className="w-full bg-gray-200 h-1.5 rounded-full overflow-hidden">
+                <div
+                  className="h-full bg-[#7CC635] rounded-full transition-all duration-700 ease-out group-hover:bg-gradient-to-r group-hover:from-[#7CC635] group-hover:to-[#5ea826]"
+                  style={{ width: `${progress}%` }}
+                ></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)